### PR TITLE
LPS-197387 Integration test for [LPS-196166]

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-197387)

### Changes

Integration test for copy action for Web Content and Stylebook. There are test already for fragments.

### CODE REVIEW

File: modules/apps/style-book/style-book-web-test/src/testIntegration/java/com/liferay/style/book/web/internal/portlet/action/test/StyleBookEntryServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+				_serviceContext);
+
+		StyleBookEntry targetStyleBookEntry =
+			_styleBookEntryService.copyStyleBookEntry(
+				_group.getGroupId(), sourceStyleBookEntry.getStyleBookEntryId(),
+				_serviceContext);
+
+		Assert.assertEquals(
+			sourceStyleBookEntry.getUserId(), targetStyleBookEntry.getUserId());
+		Assert.assertEquals(
```
Comment (ealonso) [inline modules/apps/style-book/style-book-web-test/src/testIntegration/java/com/liferay/style/book/web/internal/portlet/action/test/StyleBookEntryServiceTest.java:80]: You are checking the same 3 lines behind

File: modules/apps/style-book/style-book-web-test/src/testIntegration/java/com/liferay/style/book/web/internal/portlet/action/test/StyleBookEntryServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	@Before
+	public void setUp() throws Exception {
+		_group = GroupTestUtil.addGroup();
+
+		_serviceContext = new ServiceContext();
+
+		_serviceContext.setScopeGroupId(_group.getGroupId());
+		_serviceContext.setUserId(TestPropsValues.getUserId());
+
+		ServiceContextThreadLocal.pushServiceContext(_serviceContext);
```
Comment (ealonso) [inline modules/apps/style-book/style-book-web-test/src/testIntegration/java/com/liferay/style/book/web/internal/portlet/action/test/StyleBookEntryServiceTest.java:56]: Is it necessary? What happens if you remove it?

File: modules/apps/style-book/style-book-web-test/src/testIntegration/java/com/liferay/style/book/web/internal/portlet/action/test/StyleBookEntryServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	public void setUp() throws Exception {
+		_group = GroupTestUtil.addGroup();
+
+		_serviceContext = new ServiceContext();
```
Comment (ealonso) [inline modules/apps/style-book/style-book-web-test/src/testIntegration/java/com/liferay/style/book/web/internal/portlet/action/test/StyleBookEntryServiceTest.java:51]: I think you can just inline ServiceContextTestUtil.getServiceContext(_group) when you are calling add method

File: modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/service/test/JournalArticleServiceTest.java [hunk 4]
```diff
@@ -206,6 +208,53 @@ public void testAddArticleWithoutExternalReferenceCode() throws Exception {
 		Assert.assertEquals(_article, _latestArticle);
 	}
 
+	@Test
+	public void testCopyArticle() throws Exception {
+		String journalArticleId = StringUtil.toUpperCase(
+			RandomTestUtil.randomString());
```
Comment (ealonso) [inline modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/service/test/JournalArticleServiceTest.java:214]: You can inline it, at is should be enough if you use only RandomTestUtil.randomString(), without converting to upper case

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**